dev/properties/common.properties

```diff
@@ -31,4 +31,7 @@ org.openelisglobal.requester.phone=230 5123 1234
 
 # org.openelisglobal.providerlist.fhirstore=http://localhost:8081/fhir/
 # org.openelisglobal.requester.identifier=Practitioner/f9badd80-ab76-11e2-9e96-0800200c9a66
-facilitylist.schedule.fixedRate=864000000
\ No newline at end of file
+facilitylist.schedule.fixedRate=864000000
+
+#context for liquibase
+spring.liquibase.contexts=ci_regional
```

Comment:
@Pkom17 i gues you could comment out this property since its implementation specific

Reply:
Ok right